### DISCUSSION TITLE
How to use scripts in portable mode?

### DISCUSSION THREAD

**Discussion by parasiteoflife** [mpvnet-player/mpv.net#457] [Q&A] [answered]
I have copied the delete-current-file.cs file to `mpv.net-dir\portable_config\Scripts` and added the lines to input.conf but it doesn't work, menu does nothing, pressing the keys do nothing. Please help.  
Thanks.

**Comment by stax76** [comment, marked as answer]
C# scripts go in the `scripts-cs` folder because mpv prints a warning or error when the `scripts` folder contains files other than .lua and .js

https://github.com/stax76/mpv.net/blob/master/docs/Manual.md#c

The C# scripts are not intended for usage, they are code examples.

For delete-current-file, there is a Lua port available here:

https://github.com/stax76/mpv-scripts

**Reply by parasiteoflife** [reply]
Thank you! One question, I'm using the lua script and I'll like to know how to cancel the delete operation? When I press the first key mpv.net waits for the confirmation key but if I send "Scape" the operation isn't canceled, if I go to another file mpv.net is still waiting for the confirmation key and if I enter it the current file is deleted instead of the file that initiated the delete operation which won't be the best either as I'm looking to cancel the operation.

**Reply by stax76** [reply]
I think it's canceled automatically after 10 seconds.
